fact_extractor/plugins/unpacking/generic_carver/test/test_plugin_generic_carver_binwalk.py

```diff
@@ -16,3 +24,30 @@ def test_extraction(self):
         self.assertEqual(len(files), 1, 'file number incorrect')
         self.assertEqual(files, {'{}/64.zip'.format(self.tmp_dir.name)}, 'not all files found')
         self.assertIn('output', meta_data)
+
+
+def test_remove_false_positives_zip():
+    with TemporaryDirectory() as temp_dir:
+        test_file_zip = Path(temp_dir) / f'_fake_zip.zip.extracted' / 'fake_zip.zip'
```

Comment:
> /home/jenkins/workspace/bionic_extractor/fact_extractor/plugins/unpacking/generic_carver/test/test_plugin_generic_carver_binwalk.py:31:42: F541 f-string is missing placeholders

The f-string is not necessary here (since there is no formatting involved)